pingcap/dm
# dmctl: refine commands' usage text

### What problem does this PR solve? 

refine dmctl comands' usage text

### What is changed and how it works?

refine usage suggestion from I18N team in https://github.com/pingcap/docs-cn/pull/1878.

### Check List 

Tests 

 - Manual test (add detailed scripts or steps below)
 	1. run dmctl
 	2. observe the output of usage

Related changes

 - Need to cherry-pick to the release branch
